unit_tests/test_config.py

```diff
@@ -391,6 +391,8 @@ def test_15_new_scylla_repo(self):
         os.environ['SCT_SCYLLA_REPO'] = centos_repo
         os.environ['SCT_NEW_SCYLLA_REPO'] = centos_repo
         os.environ['SCT_USER_PREFIX'] = 'testing'
+        os.environ[
```

Comment:
wrong fix.. 
as you changed the default of `use_preinstalled_scylla` to be `True`, in this test, you need to turn it individually to `False`... because in `sct_config.py` the `type` of this parameter is `bool`, you may need to change it to be `boolean`, and then the fix as i suggest below will work, instead of this change you did here:

```
os.environ['SCT_USE_PREINSTALLED_SCYLLA'] = 'False'
```

Reply:
Israel helped me to fix this. He wrote:
`sct_config is doing it's best to make sure you won't run without the proper configuration for selecting scylla.`
`This test is checking the configuration of gce, which you changed it's default, so the test needs to provide an image (even a fake one)`
This why I added the GCE_IMAGE_DB parameter.